README.md

```diff
@@ -2,10 +2,43 @@
 
 [![License](https://img.shields.io/github/license/Qiskit/arraylias.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)
 
-**This repo is unreleased and is still in the early stages of development, there will be breaking API changes.**
+**This repo is still in the early stages of development, there will be breaking API changes**
 
-A Python library for automatic aliasing of multiple array libraries.
+Arraylias is an open-source Python library for writing Python code that
+can be run on multiple array types by aliasing different array libraries.
+Arraylias enables users to reduce the time and effort of writing codes to use various arrays.
+Arraylias provides supports for aliasing some array libraries, [NumPy](https://github.com/numpy/numpy),
+[JAX](https://github.com/google/jax), and [Tensorflow](https://github.com/tensorflow).
+
+Also, Arraylias provides another features about [Scipy](https://github.com/scipy/scipy). 
+Scipy is an open-source Python library for scientific computing. It builds on Numpy and provides support for mathematical operations, matrix calculation, and multi-dimensional arrays. Some array types allows to use scipy-like modules, which are built on the respective array libraries. Arraylias provides aliases for these scipy-like modules.
+
+Reference documentation may be found [here](https://qiskit.org/ecosystem/arraylias/), including
+[tutorials](https://qiskit.org/ecosystem/arraylias/tutorials/index.html),
+[user guide](https://qiskit.org/ecosystem/arraylias/userguide/index.html),
+[API reference](https://qiskit.org/ecosystem/arraylias/apidocs/index.html).
+
+## Installation
+
+Arraylias is installed by using `pip`:
+
+```
+pip install Arraylias
```

Comment:
FYI pip normalizes things and you don't need the capital letter. The standard project name would be:

```suggestion
pip install arraylias
```